code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -28,6 +28,14 @@
 	. = ..()
 	for(var/datum/mutation/human/HM in dna.mutations)
 		HM.on_move(src, NewLoc)
+
+	if(lying && prob(getBruteLoss()*200/maxHealth))

```

Comment:
This means only heavily brute damaged/already bleeding spessmen will leave trails, aye?

Mainly asking out of curiosity for the sake of learning. X*200/(I assume 100 by default?) where X is your brute damage and the result being used as percentage chance to decide if you leave a trail on this movement?

Less of a learning question: why 200?

Reply:
Because spessmen have 200 health total

Reply:
So break it down for me. maxHealth is 200 then, right?

Why not just use what brute loss value is returned?

Reply:
I just want to know how the magic number in the magic equation works and why.